src/display/editor/annotation_editor_layer.js

```diff
@@ -97,6 +101,9 @@ class AnnotationEditorLayer {
    * @param {number} mode
    */
   updateMode(mode = this.#uiManager.getMode()) {
+    if (mode === AnnotationEditorType.NONE) {
+      return;
+    }
```

Comment:
Sorry, but I'm not really following how this is related to the rest of the patch!?

Also, why do we no longer need to e.g. update the CSS classes (and other things) below in this method when the users switches editing off?